packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -48,9 +74,9 @@ export default  class FileInput extends UIPlugin {
     }
   }
 
-  handleInputChange (event) {
+  handleInputChange (event: Parameters<React.ChangeEventHandler<HTMLInputElement>>[0]): void {
```

Comment:
We are not using React. This should simply be `Event`.
```suggestion
  handleInputChange (event: Event): void {
```